zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/GroupEntityAddedHandler.java

```diff
@@ -66,24 +55,17 @@ public GroupEntity createNewEntity(final String id) {
   @Override
   public void updateEntity(final Record<GroupRecordValue> record, final GroupEntity entity) {
     final GroupRecordValue value = record.getValue();
-    final var entityKey = value.getEntityKey();
-
-    var memberKeys = entity.getAssignedMemberKeys();
-    memberKeys = memberKeys == null ? new HashSet<>() : memberKeys;
-    memberKeys.add(entityKey);
-
-    entity.setAssignedMemberKeys(memberKeys);
+    final var joinRelation =
+        new EntityJoinRelation()
+            .setName(IdentityJoinRelationshipType.MEMBER.getType())
+            .setParent(value.getGroupKey());
+    entity.setEntityKey(value.getEntityKey()).setJoin(joinRelation);
   }
 
   @Override
   public void flush(final GroupEntity entity, final BatchRequest batchRequest)
       throws PersistenceException {
-
-    batchRequest.updateWithScript(
-        indexName,
-        entity.getId(),
-        SCRIPT_ADD_ENTITY,
-        Map.of("addKeys", entity.getAssignedMemberKeys()));
+    batchRequest.add(indexName, entity);
```

Comment:
```suggestion
    batchRequest.addWithRouting(indexName, entity, Long.toString(entity.getJoin().getParent());
```
According to [the docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/parent-join.html#:~:text=It%20is%20required%20to%20index%20the%20lineage%20of%20a%20parent%20in%20the%20same%20shard%20so%20you%20must%20always%20route%20child%20documents%20using%20their%20greater%20parent%20id.). We need the same change in GroupEntityRemovedHandler.